### DISCUSSION TITLE
Unable to check/simulate IEEE9BUS model

### DISCUSSION THREAD

**Discussion by zyadav** [OpenModelica/OpenModelica#9356] [Q&A]
I get the following error(s), when I check and try to simulate the IEEE9BUS model. Please help. Any advice will be appreciated. I am unable to simulate any models discussed and shown in tutorials. I think I am using compatible libraries.

[1] 13:06:13 Scripting Notification
[C:/Users/syadav/AppData/Roaming/.openmodelica/libraries/index.json: 0:0-0:0]: Downloaded package index from URL https://libraries.openmodelica.org/index/v1/index.json.

[2] 13:06:21 Scripting Notification
[C:/Users/syadav/AppData/Roaming/.openmodelica/libraries//OpenIPSL 3.0.1: 0:0-0:0]: OpenIPSL 3.0.1 will not be installed since version 3.0.1 is installed.

[3] 13:06:36 Scripting Notification
Automatically loaded package OpenIPSL 3.0.1 due to uses annotation.

[4] 13:06:36 Scripting Notification
Automatically loaded package Complex 4.0.0 due to uses annotation.

[5] 13:06:36 Scripting Notification
Automatically loaded package Modelica 4.0.0 due to uses annotation.

[6] 13:06:36 Scripting Notification
Automatically loaded package ModelicaServices 4.0.0 due to uses annotation.

[7] 13:07:10 Scripting Notification
Automatically loaded package OpenIPSL 3.0.1 due to uses annotation.

[8] 13:07:10 Scripting Notification
Automatically loaded package Complex 4.0.0 due to uses annotation.

[9] 13:07:10 Scripting Notification
Automatically loaded package Modelica 4.0.0 due to uses annotation.

[10] 13:07:10 Scripting Notification
Automatically loaded package ModelicaServices 4.0.0 due to uses annotation.

[11] 13:08:00 Scripting Warning
[G:/My Drive/Downloads/DelMod/package.mo: 13:5-13:13]: Uses-annotation is missing version for library OpenIPSL. Assuming the tool-specific version="default".

[12] 13:08:00 Scripting Warning
Requested package OpenIPSL of version default, but this package was already loaded with version 3.0.1. OpenModelica cannot reason about compatibility between the two packages since they are not semantic versions.

[1] 13:08:17 Scripting Warning
[DelMod: 13:5-13:13]: [Uses-annotation is missing version for library OpenIPSL. Assuming the tool-specific version="default".](omeditmessagesbrowser:///DelMod?lineNumber=13)

[2] 13:08:17 Scripting Warning
Requested package OpenIPSL of version default, but this package was already loaded with version 3.0.1. OpenModelica cannot reason about compatibility between the two packages since they are not semantic versions.

[3] 13:08:17 Translation Warning
[OpenIPSL.Electrical.Essentials.pfComponent: 4:3-5:39]: [An inner declaration for outer component SysData could not be found and was automatically generated.](omeditmessagesbrowser:///OpenIPSL.Electrical.Essentials.pfComponent?lineNumber=4)

[4] 13:08:17 Translation Notification
[OpenIPSL.Electrical.Essentials.pfComponent: 4:3-5:39]: The diagnostics message for the missing inner is: 
    No 'System Data' component is defined.
    A default component will be used, and generate a system base of 100 MVA,
    and a frequency of 50 Hz

[5] 13:08:17 Translation Warning
[OpenIPSL.Electrical.Branches.PSAT.TwoWindingTransformer: 44:3-44:83]: [In relation twoWindingTransformer2To8.m <> 1.0, <> on Real numbers is only allowed inside functions.](omeditmessagesbrowser:///OpenIPSL.Electrical.Branches.PSAT.TwoWindingTransformer?lineNumber=44)

[6] 13:08:17 Translation Warning
[OpenIPSL.Electrical.Branches.PSAT.TwoWindingTransformer: 44:3-44:83]: [In relation twoWindingTransformer6To3.m <> 1.0, <> on Real numbers is only allowed inside functions.](omeditmessagesbrowser:///OpenIPSL.Electrical.Branches.PSAT.TwoWindingTransformer?lineNumber=44)

[7] 13:08:17 Translation Warning
[OpenIPSL.Electrical.Branches.PSAT.TwoWindingTransformer: 44:3-44:83]: [In relation twoWindingTransformer4To1.m <> 1.0, <> on Real numbers is only allowed inside functions.](omeditmessagesbrowser:///OpenIPSL.Electrical.Branches.PSAT.TwoWindingTransformer?lineNumber=44)

[8] 13:08:17 Translation Warning
[OpenIPSL.Electrical.Machines.PSAT.Order4: 25:3-30:9]: [In relation Gen1.genUnit.xq <> Gen1.genUnit.x1q, <> on Real numbers is only allowed inside functions.](omeditmessagesbrowser:///OpenIPSL.Electrical.Machines.PSAT.Order4?lineNumber=25)

[9] 13:08:17 Translation Warning
[OpenIPSL.Electrical.Machines.PSAT.Order4: 25:3-30:9]: [In relation Gen2.genUnit.xq <> Gen2.genUnit.x1q, <> on Real numbers is only allowed inside functions.](omeditmessagesbrowser:///OpenIPSL.Electrical.Machines.PSAT.Order4?lineNumber=25)

[10] 13:08:17 Translation Warning
[OpenIPSL.Electrical.Machines.PSAT.Order4: 25:3-30:9]: [In relation Gen3.genUnit.xq <> Gen3.genUnit.x1q, <> on Real numbers is only allowed inside functions.](omeditmessagesbrowser:///OpenIPSL.Electrical.Machines.PSAT.Order4?lineNumber=25)

[11] 13:08:17 Translation Error
[C:/OM119/OM64bit/OMCompiler/Compiler/NFFrontEnd/NFUnitCheck.mo: 128:5-128:91]: Internal error NFUnitCheck.checkUnits: unit check module failed

**Comment by perost** [comment]
None of the warnings or errors are unfortunately significant enough that it would cause the compilation to fail, and our [own testing](https://libraries.openmodelica.org/branches/master/OpenIPSL/OpenIPSL.html) indicates that the IEEE9 models should work. So we'll need more information to be able to help you, such as which version of OpenModelica you're using and what exact model you're trying to simulate.
